### PR TITLE
[NFV] Add MoonGen traffic generator installation

### PR DESCRIPTION
**Pull request by jlausuch**

### CODE REVIEW

**Comment by jlausuch** [comment]
@asmorodskyi can you please review this?

File: tests/nfv/moongen_installation.pm [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    zypper_call('in git gcc gcc-c++ make cmake libnuma-devel kernel-source pciutils', timeout => 300);
+
+    # Clone repository
+    assert_script_run("git clone $moongen_repo");
+
+    # Install MoonGen and dependencies
+    assert_script_run("cd MoonGen");
+    assert_script_run("bash -x build.sh",          300);
+    assert_script_run("sudo ./setup-hugetlbfs.sh", 300);
```
Comment (asmorodskyi) [inline tests/nfv/moongen_installation.pm]: you don't need to use sudo , you already in root console (line 27)

**Comment by asmorodskyi** [comment]
proof run ?

**Comment by jlausuch** [comment]
Proof run: http://10.161.8.29/tests/132

**Comment by asmorodskyi** [comment]
not exactly what you suggesting in your PR :) , 
```
 loadtest "nfv/openvswitch_dpdk";
      loadtest "nfv/vsperf_installation";
 +    loadtest "nfv/moongen_installation";
```

I want to see what will happen if you will run moongen_installation  AFTER 'openvswitch_dpdk' and 'vsperf_installation'

**Comment by jlausuch** [comment]
@asmorodskyi this is the full execution http://10.161.8.29/tests/139

**Pull request merged**